### PR TITLE
add xds model for pixiu

### PR DESCRIPTION
**Pull request by ma642**

### CODE REVIEW

**Review by ChangedenCZD — APPROVED** [review]

File: pkg/xds/model/adapter.pb.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.17.3
+// source: adapter.proto
+
+package model
+
+import (
```
Comment (ChangedenCZD) [inline pkg/xds/model/adapter.pb.go:9]: 没记错的话，导包风格应该是要分割一下的
```go
import (
	reflect "reflect"
	sync "sync"
)

import (
	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
)
```

Comment (ma642) [reply pkg/xds/model/adapter.pb.go:9]: fixed.

**Review by AlexStocks — APPROVED** [review]

File: pkg/xds/model/proto/filter.proto [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// Filter core struct, filter is extend by user
+message NetworkFilter {
+	string name = 1;
+	oneof config {
+		Config yaml = 2;
+		Config json = 3;
+		google.protobuf.Struct struct = 4;
+	}
+}
+
```
Comment (cityiron) [inline pkg/xds/model/proto/filter.proto:15]: Comments are missing

File: pkg/xds/model/proto/route.proto [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	string cluster = 1;
+	int64 cluster_not_found_response_code = 2;
+}
+
+// RouteConfiguration
+message RouteConfiguration {
+	repeated pixiu.api.v1.Router routes = 1;
+	bool dynamic = 2;
+}
+
```
Comment (cityiron) [inline pkg/xds/model/proto/route.proto:15]: ditto

**Review by ztelur — APPROVED** [review]

**Pull request merged**